.github/workflows/greet.yml

```diff
@@ -0,0 +1,50 @@
+name: 🎉 Greet First Time Contributors 🎉
+# This workflow is triggered on pull requests.
+
+on:
+  pull_request:
+    types: [opened]
+
+permissions:
+  contents: read
+
+jobs:
+  check_for_first_contribution:
+    name: '🎉 Greet First Time Contributors'
+    permissions:
+      contents: read
+      pull-requests: write # for actions/first-interaction to comment on PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1.1.1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |
+            "### Hi there, :wave:
```

Comment:
I think that, with the `|` operator, we can remove the `"` at the beginning and end of the string.